### PR TITLE
npm audit

### PR DESCRIPTION
**Pull request by txm**
Bump package versions as recommended by `npm audit`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 language: node_js
 node_js:
-  - "4"
```
Comment (knownasilya) [inline .travis.yml]: This can be 8, since 6 is EOL already.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3343 @@
+{
```
Comment (knownasilya) [inline package-lock.json]: The project uses yarn, could you remove and do `yarn` and commit that lock instead?

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "broccoli-less-single",
   "description": "Single-file-output LESS compiler for Broccoli",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (knownasilya) [inline package.json]: Could you remove as well, since usually the author does the bump once ready to release.

**Comment by gabrielgrant** [comment]
Hey @txm and @knownasilya thanks for this, and sorry it hasn't been fixed yet (thanks for the ping, Andrew). I'm actually no longer actively using this other than on one legacy project, so, while I've continued to monitor a little bit, Jason Mitchel (https://github.com/jasonmit) took over as the primary maintainer of this lib a while ago (and has done a far better job than I ever did :P ). I know he's busy with a _ton_ of projects, though, so maybe hasn't had time to show this a whole lot of love recently.

@jasonmit are you still using this/interested in maintaining it? Certainly no pressure at all either way -- we all hugely appreciate the work you've done on this over the years.

In any case, would it make sense to add @txm and/or @knownasilya as contributors? (and any chance either of you (or others) would be interested in that?)

**Comment by jasonmit** [comment]
Thanks for looping me in.  I'm still interested in maintaining broccoli-less-single, I just have notifications disabled on GitHub so I tend to miss things :)

Yes by all means feel free to add them as contributors/maintainers - I know of Ilya and have a lot of respect for his work in the Ember community.  Would love all the help 😄

File: .travis.yml (hunk 1 above)
Comment (txm) [reply .travis.yml]: 👍

File: package-lock.json (hunk 2 above)
Comment (txm) [reply package-lock.json]: 👍

File: package.json (hunk 3 above)
Comment (txm) [reply package.json]: 👍

**Comment by txm** [comment]
Thanks everyone.  PR comments updated. :)

**Pull request merged**